templates/participants.html

```diff
@@ -8,41 +8,41 @@
     <article class="col-sm-12 maincontent">
       <h1>Wszyscy uczestnicy</h1>
       <div class="table-responsive">
-        <table id="participants-table" class="table" style="width:100%!important;">
+        <table id="participants-table" class="table" style="width:100%!important;" data-order='[[ 11, "desc" ], [ 1, "asc" ]]'>
```

Comment:
Here we probably can't address the column in better way then index? :'(

Reply:
I don't think so :(